Makefile

```diff
@@ -0,0 +1,34 @@
+VERSION?=$(shell git describe --tags 2> /dev/null || echo '0.0.0')
+DOCKER_IMAGE=minimsecure/ruby-docker-image
+
+.PHONY: image
+image:
+	docker build \
+		-t minimsecure/ruby-docker-image:$(VERSION) \
+		-t $(DOCKER_IMAGE):latest \
```

Comment:
Are we currently using `latest` when pulling images? Just wondering if this is a pattern we want to use/keep using. It's convenient, but can easily propagate a breaking change.

Reply:
We are not currently using it to build retroelk:
https://github.com/violetatrium/retroelk/blob/master/.circleci/config.yml#L3

I think having `latest` is useful when developing but I agree that builds that get pushed out to any environment should not use it.